extensions/rich_text_components/Math/Math.py

```diff
@@ -32,6 +32,7 @@ class Math(base.BaseRichTextComponent):
         'schema': {
             'type': 'custom',
             'obj_type': 'MathLatexString',
+            'placeholder_text': '\\frac{x}{y}'
```

Comment:
I don't think we should make this part of the schema. Can we just use this string as the default placeholder text in math_latex_string_editor.html instead? After all, it's only needed for that one, and I don't think we'll need different placeholder texts for different math input fields.